<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageWindow</class>
 <widget class="QDialog" name="ImageWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lbl_contenu_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>360</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Contenu de la plaque :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_contenu_image">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>360</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>350</y>
     <width>651</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="contenu_horizontalLayout"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>411</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="parcourir_verticalLayout">
    <item>
     <widget class="QPushButton" name="btn_parcourir_image">
      <property name="text">
       <string>Parcourir</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl_image_frame">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lbl_mascir_image">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
